docs/Contributors_Guide/add_use_case.rst

```diff
@@ -207,58 +207,235 @@ Add Sphinx Documentation File
+
+  * METplus Workflow
+
+    * This section should begin with the init or valid times (both beginning and end), 
+      time increment, as well as any lead times being used. This should be followed 
+      by descriptions of the number of times the use case will run 
+      (which could also be inferred from the init/valid times), what each tool is doing 
+      to a level of detail sufficient for users to understand the use case workflow, 
+      and any other special notes that users should be aware of.
+      Note that if there is Python embedding, descriptions of what it accomplishes 
+      should be saved for the “Python Embedding” section. A mention of the input to 
+      the Python script and its returned dataset is sufficient for this section.
+
+  * METplus Configuration
+
+    * This section has set language that describes how all of the configuration files 
+      are loaded into METplus, followed by what’s passed in by the user at runtime, 
+      and used to produce the final configuration file that controls the METplus run. 
+      It concludes with an embedded link (and image) of the user’s configuration file 
+      the use case will run. The only two pieces that will change are the path to the 
+      use case’s configuration file, and the embedded configuration file for the use case.
+
+  * MET Configuration
+    
+    * This section has set language that describes how settings in the MET configuration file 
+      will ultimately be used to run METplus, along with any changes made from the default 
+      by the user’s configuration file. It concludes with an embedded link (and image) of 
+      the default configuration file(s) for all MET tool(s) used. Any configuration file(s)
+      listed will be hidden by default using a dropdown option. The only changes that need 
+      to occur for this section is which MET configuration file(s) is(are) embedded and the name of 
+      the dropdown.
+      If no MET tool(s) are used, that should be noted here and replace the default language.
+
+  * Python Embedding
+    
+    * This section should provide a description of any Python embedding that’s used 
+      in the use case. 
+      For a common definition, Python embedding is used to ingest a dataset not 
+      natively available for METplus intake, and results in a dataset being 
+      returned to METplus for analysis and verification purposes.
+      This section should discuss what is passed to the script from METplus, 
+      what is being done by the script, and what data structure is passed back 
+      to METplus for evaluation. The end of this section is set language 
+      directing users to review the Python requirements in the MET User’s Guide, 
+      as well as an embedded link (and image) of all Python scripts used in 
+      Python Embedding. The links to these scripts will need to be updated by 
+      the user.
+      If no Python embedding is used, that should be noted here.
+
+  * Python Scripting
```

Comment:
I'll note that `UserScript` does not have to call a Python script. It can call any shell command. Maybe this section heading should be `User Scripting` instead?

Reply:
I thought we already referred somewhere else in the documentation to any Python scripting that happens outside of METplus,  where the analysis is completed outside of the system, as "Python Scripting"?
If User Scripting is the desired nomenclature I can update it, but I want this resolved here for the foreseeable future. The naming changes on this delineation of Python scripts that pass the data in vs Python scripts that pass the statistical results back has been ongoing for way too long.

Reply:
Most of the existing UserScript examples do call a Python script because they call METdataio/METcalcpy/METplotpy Python logic. There seems to be a lot of confusion between "Python Embedding" and "Python Scripting," so using the term "User Scripting" may alleviate some of that confusion. We could note that a common example of "User Scripting" is to call a user's Python script that calls METplus Analysis Python logic.

Reply:
Please ignore this if it isn't useful.
I did a search in ReadTheDocs for "Python Scripting". It wasn't a useful search.  Even with quotes, it pulled everything with either "Python" or "Scripting".  So I did a grep from the command line instead.  

In the Main branch we haven't used "User Scripting" or "Python Scripting".  My results are listed below.

petrichor:Contributors_Guide lisag$ pwd
/Users/lisag/git/METplus/docs/Contributors_Guide

petrichor:Contributors_Guide lisag$ grep -i "Python Scripting" *.rst

petrichor:Contributors_Guide lisag$ grep -i "User Scripting" *.rst

petrichor:Contributors_Guide lisag$ grep -i "Python Embedding" *.rst
add_use_case.rst:  If the use case uses a Python embedding script, it should be
add_use_case.rst:If the use case uses a Python Embedding script or any other additional files
add_use_case.rst:caused by exceeding the memory allocation in a Python Embedding script
add_use_case.rst:If a use case utilizing Python embedding does not run successfully in 
add_use_case.rst:  the Python embedding script exceeds the Github Actions limit, and that 
continuous_integration.rst:keyword is used (see Other Keywords). A Python Embedding use case that only
continuous_integration.rst:requires the minimum packages needed to run Python Embedding can use the
continuous_integration.rst:  run a Python Embedding script. If this keyword is included with a Python
continuous_integration.rst:be added to the call to run_metplus.py so that the Python embedding script
continuous_integration.rst:for more information on how to use Python Embedding.

Reply:
Thanks for the search Lisa.

Given that we haven't officially named what this type of scripting is, I'll go ahead and rename the section User Scripting, in both the template and discussion section.